Lib/test/test_enum.py

```diff
@@ -1369,10 +1369,8 @@ class Inner(Enum):
             [Outer.a, Outer.b, Outer.Inner],
             )
 
-    @unittest.skipIf(
-            python_version < (3, 13),
-            'inner classes are still members',
-            )
+    # TODO: RUSTPYTHON
+    @unittest.expectedFailure
```

Comment:
Let's keep the original skipIf for 3.13 and add RUSTPYTHON expected failure.
Otherwise the removed lines have to be added again when we update it to 3.13 later